### PR TITLE
Fix Doc issues

### PR DESCRIPTION
**Pull request by ayshsandu**

### CODE REVIEW

File: en/docs/administer/clustering-overview.md [hunk 1]
```diff
@@ -131,3 +114,6 @@ When setting up your cluster, you must decide how you want to
 whether to front your cluster with a
 [load balancer](../../administer/load-balancing), and whether to use
 [sticky sessions](../../administer/sticky-sessions-with-manager-nodes).
+WSO2 Identity Server uses
+[Hazelcast](../../administer/configuring-hazelcast) as the underline
```
Comment (sherenewso2) [inline en/docs/administer/clustering-overview.md]: underline --> underlying

File: en/docs/learn/configuring-a-service-provider-for-adaptive-authentication.md [hunk 2]
```diff
@@ -37,10 +37,10 @@ adaptive authentication using a sample application.
 2.  Deploy the sample authenticator dependency and web application in
     WSO2 IS.
 
-    1.  Download the [org.wso2.carbon.identity.sample.extension.authenticators-5.7.0.jar](../../assets/attachments/org.wso2.carbon.identity.sample.extension.authenticators-5.7.0.jar) file and paste inside the
+    1.  Download the [org.wso2.carbon.identity.sample.extension.authenticators-5.9.0.jar](../../assets/attachments/org.wso2.carbon.identity.sample.extension.authenticators-5.9.0.jar) file and paste inside the
         `              <IS_HOME>/repository/components/dropins             ` directory.
```
Comment (sherenewso2) [inline en/docs/learn/configuring-a-service-provider-for-adaptive-authentication.md]: paste **it** inside

File: en/docs/learn/try-authorization-code-grant.md [hunk 3]
```diff
@@ -38,6 +38,14 @@ without PKCE.
 
       
     ![running-app-without-pkce](../assets/img/using-wso2-identity-server/running-app-without-pkce.png) 
+    
+    !!! Tip 
+        The playground application will invoke the **authorize**
+        endpoint of the WSO2 Identity Server in the following format.
```
Comment (sherenewso2) [inline en/docs/learn/try-authorization-code-grant.md]: in the following format --> using the following format

File: en/docs/learn/self-registration-and-account-confirmation.md [hunk 4]
```diff
@@ -220,7 +220,24 @@ Next, you can try out self-registration.
 
 3.  Fill in the user details, provide consent to share the requested
     information and then click **Register**.
+    ![self-signup-form](../assets/img/using-wso2-identity-server/self-signup-form.png) 
+        
+    ??? Abstract "Click here to see how to configure above requested attributes for the self registration"
```
Comment (sherenewso2) [inline en/docs/learn/self-registration-and-account-confirmation.md]: Click to see steps on configuring requested attributes for self registration

File: en/docs/learn/self-registration-and-account-confirmation.md [hunk 5]
```diff
@@ -220,7 +220,24 @@ Next, you can try out self-registration.
...
+    ![self-signup-form](../assets/img/using-wso2-identity-server/self-signup-form.png) 
+        
+    ??? Abstract "Click here to see how to configure above requested attributes for the self registration" 
+        
+        -   Attributes that shows in the
+        self sign up page are the WSO2 [local dialect's](../../learn/adding-claim-mapping/#add-local-claim) claims which has
+        **Supported by Default** configuration enabled. 
+        -   Navigate to claim
+        dialect list page and go to **http://wso2.org/claims** dialect and
+        expand any claim to view the configuration.
```
Comment (sherenewso2) [inline en/docs/learn/self-registration-and-account-confirmation.md]: Click **List** under **Claims** and select the **http://wso2.org/claims** dialect. Expand any claim to view the configuration.

File: en/docs/learn/self-registration-and-account-confirmation.md [hunk 6]
```diff
@@ -220,7 +220,24 @@ Next, you can try out self-registration.
...
+        
+        -   Attributes that shows in the
+        self sign up page are the WSO2 [local dialect's](../../learn/adding-claim-mapping/#add-local-claim) claims which has
+        **Supported by Default** configuration enabled. 
+        -   Navigate to claim
+        dialect list page and go to **http://wso2.org/claims** dialect and
+        expand any claim to view the configuration. 
+        -   Mandatory attributes of
+        the self sign-up page are the claims that has **Required**
+        configuration enabled.
```
Comment (sherenewso2) [inline en/docs/learn/self-registration-and-account-confirmation.md]: that has --> that have the

File: en/docs/learn/self-registration-and-account-confirmation.md [hunk 7]
```diff
@@ -220,7 +220,24 @@ Next, you can try out self-registration.
 
 3.  Fill in the user details, provide consent to share the requested
     information and then click **Register**.
+    ![self-signup-form](../assets/img/using-wso2-identity-server/self-signup-form.png) 
+        
+    ??? Abstract "Click here to see how to configure above requested attributes for the self registration" 
+        
+        -   Attributes that shows in the
+        self sign up page are the WSO2 [local dialect's](../../learn/adding-claim-mapping/#add-local-claim) claims which has
```
Comment (sherenewso2) [inline en/docs/learn/self-registration-and-account-confirmation.md]: The attributes that show up on the self sign up page are WSO2 [local dialect](../../learn/adding-claim-mapping/#add-local-claim) claims that have the **Supported by Default** configuration enabled.

File: en/docs/administer/configuring-hazelcast.md [hunk 8]
```diff
@@ -44,4 +44,22 @@ important to add all the members to the well-known members list in the `
      [clustering]
      ...
      members = ["<member-x-host>:<member-x-port>", ...]
-     ```
\ No newline at end of file
+     ```
+     
+!!! info
+    WSO2 provides Hazelcast Community Edition as its default
```
Comment (YvonneW) [inline en/docs/administer/configuring-hazelcast.md]: WSO2 Identity Server provides Hazelcast Community ....

File: en/docs/administer/configuring-hazelcast.md [hunk 9]
```diff
@@ -44,4 +44,22 @@ important to add all the members to the well-known members list in the `
...
+    clustering engine. For clustering on a secure channel (i.e., secure
+    Hazelcast), use Hazelcast Enterprise. To integrate with Hazelcast
+    Enterprise, there are provisions to provide license key under clustering
+    configurations. Advanced users can fine-tune Hazelcast by creating a
+    `         <IS_HOME>/repository/conf/hazelcast.properties        `
```
Comment (YvonneW) [inline en/docs/administer/configuring-hazelcast.md]: by creating a `hazelcast.properties` file in the `<IS_HOME>/repository/conf` directory

File: en/docs/administer/configuring-hazelcast.md [hunk 10]
```diff
@@ -44,4 +44,22 @@ important to add all the members to the well-known members list in the `
...
+    file and adding the relevant Hazelcast properties as described in the
+    [Hazelcast Advanced Configuration Properties
+    documentation](https://docs.hazelcast.org/docs/3.0/manual/html/ch12s06.html).
+    
+    Add the following property to `hazelcast.properties` file to add the
+    license key of Hazelcast Enterprise subscription.
```
Comment (YvonneW) [inline en/docs/administer/configuring-hazelcast.md]: To add the Hazelcast Enterprise subscription license key, add the following property to the  `hazelcast.properties` file.

File: en/docs/develop/carbon-remote-user-store-manager.md [hunk 11]
```diff
@@ -42,3 +42,8 @@ following steps.
 
 3.  Update other fields as required (description is give for each
     property) and click **Add**.
+
+4. Import the public certificate of the remote server to primary
+   IS(IS1)'s
+   `<IS_HOME>/repository/resources/security/client-truststore.jks` file
+   and restart the server.
```
Comment (YvonneW) [inline en/docs/develop/carbon-remote-user-store-manager.md]: ... to the primary WSO2 Identity Server instance's (IS1) truststore (`client-truststore.jks`) that is in the `<IS_HOME>/repository/resources/security` directory and restart WSO2 Identity Server.

File: en/docs/get-started/quick-start-guide.md [hunk 12]
```diff
@@ -314,19 +314,21 @@ Let's use the command-line to check the MFA functionality.
...
+   1.  Download the [org.wso2.carbon.identity.sample.extension.authenticators-5.9.0.jar](../../assets/attachments/org.wso2.carbon.identity.sample.extension.authenticators-5.9.0.jar) file and paste inside the
         `              <IS_HOME>/repository/components/dropins             ` directory.
 
-   2.  Download the [sample-auth.war](../../assets/attachments/sample-auth.war) file and paste it inside the `             <IS_HOME>/repository/deployment/server/webapps            `
+   2.  Download the [sample-auth.war](https://github.com/wso2/samples-is/releases/download/v4.1.0/sample-auth.war) file and paste it inside the `             <IS_HOME>/repository/deployment/server/webapps            `
         folder.  
         This `             .war            ` file contains the WEB UI
         for the sample authenticators used in this tutorial.
 
    3.Add the followings to the deployment.toml in <IS_HOME>/repository/conf
```
Comment (YvonneW) [inline en/docs/get-started/quick-start-guide.md]: `deployment.toml` file in the `<IS_HOME>/repository/conf` directory.

File: en/docs/learn/kerberos-grant.md [hunk 13]
```diff
@@ -74,15 +74,15 @@ Kerberos ticket for an OAuth2 token.
 
 Follow the instructions below to configure Kerberos Grant with WSO2 IS:
 
-1.  Download the [kerberos grant.jar](../assets/attachments/org.wso2.carbon.identity.sample.extension.authenticators-5.7.0.jar) file.
+1.  Download the [kerberos grant.jar](../assets/attachments/kerberos-grant-1.0.0.jar) file.
 2.  Copy the JAR into the `<IS_HOME>/repository/components/lib` directory.
-3.  Add the following entry to the `deployment.toml` file in the `<IS_HOME>/repository/conf/` folder.
+3.  Add the following entry to the `deployment.toml` file in the
+    `<IS_HOME>/repository/conf/` folder to enable **Kerberos** grant
+    type.
```
Comment (YvonneW) [inline en/docs/learn/kerberos-grant.md]: To enable the **Kerberos** grant, add the following entry to the `deployment.toml` file in the `<IS_HOME>/repository/conf/` folder.

File: en/docs/learn/openid-connect-scopes-and-claims.md [hunk 14]
```diff
@@ -41,30 +41,41 @@ under **Manage**. Click **Add** to add a new scope mapping or click
     can add and remove claims from the scope by using the **Add claims**
     and **Update** buttons respectively. You can also delete a scope
     claim mapping.
-
+    ![oidc-scope-list](../assets/img/using-wso2-identity-server/oidc-scope-list.png)
+    
+!!! Tip 
+    You can add custom claims to any OIDC scope by following steps;
```
Comment (YvonneW) [inline en/docs/learn/openid-connect-scopes-and-claims.md]: To add custom claims to any OIDC scope:

File: en/docs/learn/self-registration-and-account-confirmation.md [hunk 15]
```diff
@@ -16,7 +16,7 @@ delete such accounts to manage resources better.
 The following sections walk you through configuring and trying out
 self-registration.
     
-!!! Warning 
+??? Warning "Click to see instructions specific for a migrated deployment" 
     If you have migrated from a previous IS version, ensure that the `IdentityMgtEventListener` with the
```
Comment (YvonneW) [inline en/docs/learn/self-registration-and-account-confirmation.md]: WSO2 Identity Server version

File: en/docs/learn/openid-connect-scopes-and-claims.md [hunk 16]
```diff
@@ -41,30 +41,41 @@ under **Manage**. Click **Add** to add a new scope mapping or click
...
+    
+    1.  Add [a custom claim to the local dialect](../../learn/adding-claim-mapping/#add-local-claim), `https://wso2.org/claims`
+    2.  Add [an external claim](../../learn/adding-claim-mapping/#add-external-claim) into the `http://wso2.org/oidc/claim` dialect referring the local claim created in step1.
+    3.  In the **OIDC Scopes** section, add the newly added OIDC claim under the desired *scopes*.
+
```
Comment (YvonneW) [inline en/docs/learn/openid-connect-scopes-and-claims.md]: For a sample use case, see [Handling Custom OIDC Claims](https://medium.com/identity-beyond-borders/handling-custom-claims-with-openid-connect-in-wso2-identity-server-56d3b6e4319b).

Comment (ayshsandu) [reply en/docs/learn/openid-connect-scopes-and-claims.md]: Fixed.

File: en/docs/learn/self-registration-and-account-confirmation.md (hunk 15 above)
Comment (ayshsandu) [reply en/docs/learn/self-registration-and-account-confirmation.md]: Fixed.

File: en/docs/learn/openid-connect-scopes-and-claims.md (hunk 14 above)
Comment (ayshsandu) [reply en/docs/learn/openid-connect-scopes-and-claims.md]: Fixed.

**Pull request merged**